util/gather-docs.ts

```diff
@@ -0,0 +1,103 @@
+import { readdir, stat } from "fs/promises";
+import * as path from "path";
+import { spawn } from "child_process";
+
+function convertWindowsToLinuxPath(windowsPath) {
+    // Normalize the path to handle mixed slashes
+    const normalizedPath = path.normalize(windowsPath);
+    // Convert to POSIX format
+    return path.posix.join(...normalizedPath.split(path.sep)).split("G:/").join("/mnt/g/");
+}
+
+async function convertDocBookToMarkdown(inputFile: string): Promise<string> {
+    return new Promise((resolve, reject) => {
+        const pandoc = spawn("wsl", ["pandoc", "-f", "docbook", "-t", "markdown", "-s", convertWindowsToLinuxPath(inputFile)]);
```

Comment:
Where does pandoc come from?

Reply:
`apt install pandoc`, Ideally the docs.vecdb file would be generated as part of CI / Release building, but it really needs a GPU...

Reply:
Also it should really be done in the HPCC-Platform repo.
FWIW the code in the "/utils" folder is only for periodic execution and the output of those runs get committed to GitHub - similar to how we do the wsdl stuff in the Viz comms layer.